applications/CompressiblePotentialFlowApplication/python_scripts/compute_lift_process.py

```
@@ -67,18 +67,7 @@ def _CalculateWakeTangentAndNormalDirections(self):
         free_stream_velocity = self.fluid_model_part.ProcessInfo.GetValue(CPFApp.FREE_STREAM_VELOCITY)
         self.free_stream_velocity_norm = free_stream_velocity.norm_2()
         self.wake_direction = free_stream_velocity / self.free_stream_velocity_norm
-
-        self.wake_normal = KratosMultiphysics.Vector(3)
-        if(self.fluid_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE] == 2):
-            self.wake_normal[0] = -self.wake_direction[1]
-            self.wake_normal[1] = self.wake_direction[0]
-            self.wake_normal[2] = 0.0
-        elif(self.fluid_model_part.ProcessInfo[KratosMultiphysics.DOMAIN_SIZE] == 3):
-            # TODO: Read wake normal from wake process
-            self.wake_normal[0] = 0.0
-            self.wake_normal[1] = 0.0
-            self.wake_normal[2] = 1.0
-
+        self.wake_normal = self.fluid_model_part.ProcessInfo[CPFApp.WAKE_NORMAL]
```

I'd throw a meaningful error in case we have not set it (i.e., in case we didn't call the wake process). Maybe we can simply check if the norm is (close to) zero.

---

I sill make another PR for this.